### ISSUE TITLE
Computing temporal betweenness and Closeness 

### ISSUE THREAD

**Issue by MarwanG** [IngoScholtes/pathpy#20]
Hello,

I am trying to do a comparison study about existing temporal betweenness and closeness methods. 
However, whenever I run the method GetTemporalBetweenness I am getting this error, 
minD, minPaths = Distances.GetMinTemporalDistance(t, delta=1, collect_paths=True)
NameError: name 'Distances' is not defined. 
I am unable to find the module Distances in any of your github depots. I am wondering if this part of the code is somewhere ? 

The same error happens also for closeness.

Regards,
Marwan

**Comment by IngoScholtes** [comment]
Hi Marwan 

Thanks for noticing. This is the result of a refactoring gone wrong. The original methods were implemented in pyTempNet (see https://github.com/IngoScholtes/pyTempNets/blob/master/pyTempNet/Paths.py#L188). I will try to adjust them to implement a quick patch. At the same time, we are preparing pathpy 2.0, which comes with a largely improved implementation of temporal centralities.

Please stay tuned. 

Kind regards 

Ingo

**Comment by IngoScholtes** [comment]
Hi Marwan 

Sorry for the delay, I finally had time to look into this. 

The simple solution to this issue is, that these methods should not be there in the first place. They are a remainder of the previous library pyTempNet. In pathpy, they have been replaced by corresponding methods on the Paths object, see [here](https://github.com/IngoScholtes/pathpy/blob/master/pathpy/Paths.py#L984) and [here](https://github.com/IngoScholtes/pathpy/blob/master/pathpy/Paths.py#L1014), which should be working just fine.

The key idea is the following: Temporal betweenness and closeness centrality are defined based on shortest time-respecting paths in a temporal network. And we can use the Paths.fromTemporalNetwork method to extract tinme-respecting paths from a temporal network instance (defining a max. temporal distance delta for the path calculation). This is why we moved the centrality calculation to the Paths object. 

We thus removed the old methods, as they will cause confusion. Hope that helps. Let me know if you have any further questions. 

Kind regards 

Ingo
